includes/class-wc-subscriptions-email.php

```diff
@@ -366,11 +366,7 @@ public static function attach_woocommerce_transactional_email( $hook = '', $prio
 			$hook = current_filter();
 		}
 
-		if ( false === wcs_is_woocommerce_pre( '3.0' ) && apply_filters( 'woocommerce_defer_transactional_emails', false ) ) {
-			add_action( $hook, array( 'WC_Emails', 'queue_transactional_email' ), $priority, $accepted_args );
-		} else {
-			add_action( $hook, array( 'WC_Emails', 'send_transactional_email' ), $priority, $accepted_args );
-		}
+		add_action( $hook, array( 'WC_Emails', 'send_transactional_email' ), $priority, $accepted_args );
```

Comment:
I think the update here this is incorrect. `wcs_is_woocommerce_pre( '3.0' )` will always be `false` right? ie no one will ever be running pre 3.0

```php
if ( apply_filters( 'woocommerce_defer_transactional_emails', false ) ) {
	add_action( $hook, array( 'WC_Emails', 'queue_transactional_email' ), $priority, $accepted_args );
} else {
	add_action( $hook, array( 'WC_Emails', 'send_transactional_email' ), $priority, $accepted_args );
}
```